eam-light-backendejb/src/main/java/ch/cern/cmms/eamlightejb/MonitoringService/MonitoringService.java

```diff
@@ -21,56 +20,79 @@
     @Inject
     private InforClient inforClient;
 
-    private String EQUIPMENT_CODE = "TEST_MONITORING";
-    private String WORKORDER_CODE = "25316908";
     private SimpleDateFormat simpleDateFormat = new SimpleDateFormat("dd-MMM-yyyy HH-mm-ss", Locale.ENGLISH);
+    private Map<String, String> responses;
 
-    public Map<String, String> monitoring(String equipment, String number, InforContext inforContext) {
-        Map<String, String> responses = new HashMap<>();
-        Equipment equipmentForUpdate = new Equipment();
-        WorkOrder workOrderForUpdate = new WorkOrder();
-
+    public Map<String, String> monitorEndpoints(String equipment, String number,
+        String equipmentNumber, String workorderNumber,
+        InforContext inforContext) {
+        responses = new HashMap<>();
         try {
-            String result = inforClient.getEquipmentFacadeService().readEquipment(inforContext,
-                equipment).toString();
+            String result = monitoringReadEquipment(equipment,
+                inforContext).toString();
             responses.put("READEQUIPMENT", result);
-        } catch (InforException e) {
+        } catch (Exception e) {
             responses.put("READEQUIPMENT", "ERROR " + e.getMessage());
         }
         try {
-            String result = inforClient.getWorkOrderService().readWorkOrder(inforContext, number).toString();
+            String result = monitoringReadWorkorder(number,
+                inforContext).toString();
             responses.put("READWORKORDER", result);
         } catch (Exception e) {
             responses.put("READWORKORDER", "ERROR " + e.getMessage());
         }
         try {
-            equipmentForUpdate.setCode(EQUIPMENT_CODE);
-            equipmentForUpdate.setDescription("MONITORING ASSET / " + getCurrentDate());
-
-            inforClient.getEquipmentFacadeService().updateEquipment(inforContext, equipmentForUpdate);
-            String result =
-                inforClient.getEquipmentFacadeService().readEquipment(inforContext, equipmentForUpdate.getCode())
-                    .toString();
+            String result = monitoringUpdateEquipment(equipmentNumber,
+                inforContext).toString();
             responses.put("UPDATEEQUIPMENT", result);
         } catch (Exception e) {
-            responses.put("UPDATEEQUIPMENT", e.getMessage());
+            responses.put("UPDATEEQUIPMENT", "ERROR " + e.getMessage());
```

Comment:
"ERROR " should be put into a constant. If this changes, we shall need to only change it in one place